mk/config.mk

```diff
@@ -442,6 +442,7 @@ $(call force,CFG_DT,y)
 endif
 CFG_EMBED_DTB ?= n
 CFG_DT ?= n
+CFG_EXTERNAL_DT ?= $(CFG_DT)
```

Comment:
Please add a description of this in the comment above.

Reply:
Added a description in the comment